src/lightning_app/core/api.py

```diff
@@ -70,21 +69,18 @@ class SessionMiddleware:
 
 
 class UIRefresher(Thread):
-    def __init__(self, api_publish_state_queue, api_response_queue, refresh_interval: float = 0.1) -> None:
+    def __init__(self, api_publish_state_queue, api_response_queue) -> None:
         super().__init__(daemon=True)
         self.api_publish_state_queue = api_publish_state_queue
         self.api_response_queue = api_response_queue
         self._exit_event = Event()
-        self.refresh_interval = refresh_interval
 
     def run(self):
         # TODO: Create multiple threads to handle the background logic
         # TODO: Investigate the use of `parallel=True`
         try:
             while not self._exit_event.is_set():
                 self.run_once()
-                # Note: Sleep to reduce queue calls.
-                sleep(self.refresh_interval)
```

Comment:
How will we address the increase in queue calls?

Reply:
Answer in private.

Reply:
@awaelchli are you fine with it? :)